charts/tidb-cluster/templates/config/_pump-config.tpl

```diff
@@ -26,3 +26,17 @@ pd-urls = "http://{{ .Values.clusterName }}-pd:2379"
 # ssl-cert = "/path/to/drainer.pem"
 # Path of file that contains X509 key in PEM format for connection with cluster components.
 # ssl-key = "/path/to/drainer-key.pem"
+#
+#
+# we suggest using the default config of the embedded LSM DB now, do not change it useless you know what you are doing
```

Comment:
useless -> unless

Reply:
Nice catch! This is copied from tidb-binlog repo, you can also send a PR there for the typo :-)